thesis-de/content-de.tex

```diff
@@ -1754,6 +1753,7 @@ \subsubsection{Durchführung Evaluierung Frontend}
   Negativ:
   \1 Keine Möglichkeit, leicht Plugins hinzuzufügen
   \1 Webdesign kann nicht angepasst werden
+  \1 Besteht aus mehreren Diensten, wie graphite-web und carbon-cache
```

Comment:
sorry, ich meinte nicht unter positiv, sondern in der allgemeinen erklärung zu graphite.